packages/@aws-cdk/toolkit-lib/lib/api/io/private/messages.ts

```diff
@@ -107,10 +108,10 @@ export const warn = <T>(message: string, code?: VALID_CODE, payload?: T) => {
 /**
  * Creates an info level message.
  */
-export const info = <T>(message: string, code?: VALID_CODE, payload?: T) => {
+export const info = <T>(message: string, code?: CodeInfo, payload?: T) => {
```

Comment:
i forgot how we said we wanted to soft enforce that we call `info` on a code with level `info`. can you jog my memory @mrgrain ?